From 5c55c0e5c220ab99ce8bae96c1527ccefdf1899e Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 19 Mar 2020 22:27:50 +0200
Subject: [PATCH] [SW-10047] pll: udelay: remove namespace prefix

Remove the namespace from udelay API for simplicity.
This way a caller can benefit the shorter and descriptive enough
udelay() function calls.

Change-Id: Ib519964da440794f8257d75d9fe77b7e85161125
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/app/preboot.c b/app/preboot.c
index b281413..e29a3d0 100644
--- a/app/preboot.c
+++ b/app/preboot.c
@@ -150,7 +150,7 @@
 	 * Note: we assume CPU frequency is not changed while CPU is running,
 	 * so the udelay init is done once before we start.
 	 */
-	hl_pll_udelay_init();
+	udelay_init();
 
 	hl_uart_init();
 	preboot_version();
diff --git a/common/pll.c b/common/pll.c
index 14de247..1241162 100644
--- a/common/pll.c
+++ b/common/pll.c
@@ -28,12 +28,12 @@
 
 static u64 cycles_per_usec = BYPASS_CYCLES_PER_USEC;
 
-void hl_pll_udelay_init(void)
+void udelay_init(void)
 {
 	cycles_per_usec = hl_pll_get_cpu_cycles_per_usec();
 }
 
-void hl_pll_udelay(unsigned int usec)
+void udelay(unsigned int usec)
 {
 	u64 cnt = (cycles_per_usec * usec) / MIN_CYCLES_PER_ITER;
 
diff --git a/common/pll.h b/common/pll.h
index d567bdb..e1306ee 100644
--- a/common/pll.h
+++ b/common/pll.h
@@ -27,9 +27,9 @@
 	HL_PLL_DIV_SIZE,
 };
 
-void hl_pll_udelay_init(void);
+void udelay_init(void);
 u64 hl_pll_get_cpu_cycles_per_usec(void);
 u32 hl_pll_get_freq(u32 pll_base, enum hl_pll_div_out div);
-void hl_pll_udelay(unsigned int usec);
+void udelay(unsigned int usec);
 
 #endif /* _HL_PLL_H_ */
diff --git a/gaudi/common.c b/gaudi/common.c
index 1871fe5..b1d9952 100644
--- a/gaudi/common.c
+++ b/gaudi/common.c
@@ -79,12 +79,12 @@
 	regmap_writel(0x00402801, mmPCIE_DBI_GEN3_RELATED_OFF);
 
 	/* delay before executing Synopsys W/A > 300us */
-	hl_pll_udelay(300);
+	udelay(300);
 
 	/* Execute the Synopsys W/A */
 	regmap_writel(0xF, mmPCIE_PHY_CR_PARA_CTRL);
 	regmap_writel(0x0004A07D, mmPCIE_PHY_CR_PARA_CMD);
-	hl_pll_udelay(20);
+	udelay(20);
 
 	/* Enable and Mask LTSSM (erratum 102) */
 	regmap_writel(3, mmPCIE_AUX_LTSSM_EN);
diff --git a/goya2/common.c b/goya2/common.c
index 062e399..98c26a1 100644
--- a/goya2/common.c
+++ b/goya2/common.c
@@ -81,12 +81,12 @@
 	regmap_writel(0x00402801, mmPCIE_DBI_GEN3_RELATED_OFF);
 
 	/* delay before executing Synopsys W/A > 300us */
-	hl_pll_udelay(300);
+	udelay(300);
 
 	/* Execute the Synopsys W/A */
 	regmap_writel(0xF, mmPCIE_PHY_CR_PARA_CTRL);
 	regmap_writel(0x0004A07D, mmPCIE_PHY_CR_PARA_CMD);
-	hl_pll_udelay(20);
+	udelay(20);
 
 	/* Enable and Mask LTSSM (erratum 102) */
 	regmap_writel(3, mmPCIE_AUX_LTSSM_EN);
diff --git a/goya2_arc/common.c b/goya2_arc/common.c
index e713727..454451e 100644
--- a/goya2_arc/common.c
+++ b/goya2_arc/common.c
@@ -85,12 +85,12 @@
 	regmap_writel(0x00402801, mmPCIE_DBI_GEN3_RELATED_OFF);
 
 	/* delay before executing Synopsys W/A > 300us */
-	hl_pll_udelay(300);
+	udelay(300);
 
 	/* Execute the Synopsys W/A */
 	regmap_writel(0xF, mmPCIE_PHY_CR_PARA_CTRL);
 	regmap_writel(0x0004A07D, mmPCIE_PHY_CR_PARA_CMD);
-	hl_pll_udelay(20);
+	udelay(20);
 
 	/* Enable and Mask LTSSM (erratum 102) */
 	regmap_writel(3, mmPCIE_AUX_LTSSM_EN);
